src/test/java/org/folio/rest/utils/TestResourceClient.java

```diff
@@ -0,0 +1,65 @@
+package org.folio.rest.utils;
+
+import static org.folio.rest.impl.APITests.getOkapiUrl;
+
+import org.folio.HttpStatus;
+
+import io.restassured.response.Response;
+
+public class TestResourceClient {
```

Comment:
Could this be called `ResourceClient` instead? Is it important that this is only for tests, given that it is defined within the test code source set?

Reply:
It is renamed now. The `Test` part was just to not mix up production client with the test client.